keps/sig-node/NNNN-pass-resource-reqs-limits-to-cri/README.md

```diff
@@ -68,9 +68,10 @@ This KEP proposes a CRI API extension for disclosing container
 
 Currently, kubelet is (only) responsible for selecting CPUs and
 memories to be allocated for a container by sending
-`UpdateContainerResources` request to CRI. Non-native resources such
-as extended resources and the device plugin resources are not visible
-to container runtimes.
+`UpdateContainerResources` request to CRI. And, even there the original details
```

Comment:
Ah yes, bad wording. I actually dunno about cpumanager. But cpumanager might be disabled in which case there is no pinning (AFAIU). Changed the wording to `may be lost`.

Reply:
Sorry, my bad. That's not what I meant. The 'original details of the resource spec is lost' statement is very true IMO.

My nitpick was about whether we correctly state how `kubelet` uses `CRI` for container resource management (only `UpdateContainerResources`). Or IOW if it is really still so that the initial container creation happens *without* any CPU/memory pinning which is then fixed up by the `reconcilation loop` in `CPUManager`.

I just checked it and this stupid oddity should be finally fixed in `1.21`. The recently added [`PreCreateContainer()`](https://github.com/kubernetes/kubernetes/blob/b8766230700f8906daeaa1654497691429349311/pkg/kubelet/cm/internal_container_lifecycle_linux.go#L29) lifecycle call tries to correctly fill in `CpusetCpus` and `CpusetMems` if CPUManager is enabled.

So this should (ideally) be rephrased. Maybe something like

Currently kubelet is (only) responsible for selecting CPUs and memories to be
allocated for a container. This is done by setting up LinuxContainerResources
in the corresponding `CreateContainer` and `UpdateContainerResources`
requests according to this selection.

But as I stated initially, in the big picture this is rather a nitpick.

Reply:
OK 😄 Did some rewording and updated the PR (with a new patch)